### DISCUSSION TITLE
Can we set the own Voice channel in the settc command?

### DISCUSSION THREAD

**Discussion by vico93** [jagrosh/MusicBot#1182] [Questions] [answered]
I'm asking because Discord [is rolling out a new feature where a Voice channel will have a "embbeded" text channel inside itself](http://dis.gd/text-in-voice).

Being able to restrict the bot's commands to the very voice channel that it is joined would be very interesting feature to have.

**Comment by MichailiK** [comment, marked as answer]
JDA, the library we use to connect to Discord, doesn't support text in voice yet. Once JDA adds support, we will very likely support it in the next update as well.

**Reply by vico93** [reply]
Okay, thanks a lot for this info!
